pkg/arvo/lib/link-store.hoon

```diff
@@ -1,7 +1,10 @@
 ::  link: social bookmarking
```

Comment:
I don't think I understand why this entire file was renamed from `/lib/link` to `/lib/link-store`. Makes sense for the json conversions, but the other functions here mostly operate on suite-wide data, rate?

Reply:
none of these are used outside the link store though. imo the idea of 'suite-wide' data is kinda vague because the store itself should always store the 'suite-wide' data

Reply:
You're right that it is a bit vague. My thinking there is that, even if you're interacting only with one of the hooks, you're still pulling in the core data types and working with those. Pulling in the store lib for that (which could very well contain helpers for interacting with the store) makes is slightly less obvious what you're actually touching. But it's also not the biggest deal.